kratos/processes/apply_periodic_boundary_condition_process.h

```diff
@@ -0,0 +1,196 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Aditya Ghantasala
+//
+//
+
+#ifndef APPLY_PERIODIC_CONDITION_PROCESS_H
+#define APPLY_PERIODIC_CONDITION_PROCESS_H
+
+// System includes
+#include <string>
+#include <iostream>
+#include <chrono>
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "geometries/point.h"
+#include "processes/process.h"
+#include "utilities/math_utils.h"
+#include "includes/kratos_parameters.h"
+#include "includes/dof.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+
+class ApplyPeriodicConditionProcess : public Process
```

Comment:
you might need to add KRATOS_API here to get it compile in Win

Reply:
done